ports/ftxui/portfile.cmake

```
@@ -6,6 +6,8 @@ vcpkg_from_github(
     REF c033ca61ae8fb264542d0326d1309e0a3bde945a # 3.0.0
     SHA512 ca1468f30f90c3a886fbb6dea113699623d601da10b39a6a33d89780cd825bec8deb431872a2515fce05c7a5d581d4b56860b19654df8cf90e389dfa964f013c
     HEAD_REF master
+    PATCHES
+        fix_clang_crash-094d8d9d.patch

```

Please add the comment for this patch, when updating this port to new version, this patch can be removed.
```suggestion
        fix_clang_crash-094d8d9d.patch #Upstream PR https://github.com/ArthurSonzogni/FTXUI/pull/421
```

---

done!